src/sortStudents.ts

```diff
@@ -1,16 +1,55 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function calculateAvarageMark(allMarks: number[]): number {
```

Comment:
We don't need an explicit return type for the primitives.
```suggestion
function calculateAvarageMark(allMarks: number[]) {
```

Reply:
i think it`s required by eslint, as otherwise it shows "Missing return type on function.  eslint"